### ISSUE TITLE
Summernote not showing

### ISSUE THREAD

**Issue by elmartino** [martinsvb/ng2-summernote#6]
Hi, 

I followed the readme, got no errors, but summernote is not showing up. 
Only thing created is an empty `<div class="summernote"></div>` inside the `<ng2-summernote>`.

Any ideas?
Thanks!

**Comment by jeetenparmar** [comment]
Did you get any solution ?

**Comment by elmartino** [comment]
Nope unfortunately not. I used the non angular2 version of summernote for the time being..

**Comment by jeetenparmar** [comment]
which one ? Can you give me any example ?

**Comment by tewson** [comment]
Hi, I assume you initialised the `ngModel` attribute of the `ng2-summernote` component with an empty string (i.e. `''`)?

If that's the case, it seems like the condition on [this line](https://github.com/martinsvb/ng2-summernote/blob/master/ng2-summernote.ts#L158) treats the empty string as a falsy value so the editor  is not rendered.

A workaround for this is initialising the `ngModel` attribute of the `ng2-summernote` component with a truthy empty string by using `new String('')` instead of just `''`.

Hope this helps.

**Comment by kakalos12** [comment]
@martinsvb Do you have any plans to update this ?

**Comment by segvan** [comment]
yes there is a bug in writeValue(value: any) method. Instead of if (value) { ... } should be if (value !== 'undefined' && value !== null) { ... }

**Comment by martinsvb** [comment]
I think you can kindly use if (!!value) {}
But, I didn't support this package.
Can somebody make PR?

Thank you

**Comment by segvan** [comment]
No (!!value) will not work because empty string will still give false.
I will do PR with pleasure.
